src/utils.rs

```diff
@@ -102,7 +102,7 @@ mod tests {
 
     #[test]
     fn exec_with_output_stdout() {
-        let result = internal_exec_cmd("/bin/echo", &["-n", "hello"]);
+        let result = internal_exec_cmd("/bin/sh", &["-c", "echo -n hello"]);
         let expected = Some(CommandOutput {
             stdout: String::from("hello"),
```

Comment:
and this it should fix it.
```suggestion
             stdout: String::from("hello\n"),
```